packages/kit/src/views/Developer/pages/Gallery/Components/stories/CryptoGallery.tsx

```
@@ -0,0 +1,715 @@
+/* eslint-disable spellcheck/spell-checker */
+/* eslint-disable prefer-const */
+import { useState } from 'react';
+
+import crypto from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)